index.bs

```diff
@@ -263,6 +263,34 @@ so that it works across screen sizes,
 and allows users to choose their own font and font size without causing text to overflow.
 </div>
 
+<h3 id="backwards-compat">Support Existing Content</h3>
+
+Existing content often relies upon expected user agent processing and behavior to function as intended.
+Processing requirements should be specified to ensure that user agents will be able to handle most existing content.
+In particular, it should be possible to process existing web content in current user agents and get results that are compatible with the existing expectations of users and authors,
+based on the behavior of existing user agents.
+
+Content relying on existing user agent behavior can take many forms, such as:
+
+- Relying on features that are entirely proprietary.
```

Comment:
Perhaps an additional bullet point here describing specified and widely implemented features. The rest mostly talks about un(der)specified behavior, and it seems like we're missing the main thing.